ListingService/Sources/Models/Listing.swift

```diff
@@ -19,13 +20,19 @@ public class Listing: Decodable {
         case url
         case fullServerID = "name"
         case thumbnailURL = "thumbnail"
+        case displayName = "display_name"
     }
 
     public required init(from decoder: Decoder) throws {
         let values = try decoder.container(keyedBy: CodingKeys.self)
         let innerData = try values.nestedContainer(keyedBy: InnerDataKeys.self, forKey: .innerData)
         self.title = try innerData.decode(String.self, forKey: .title)
         self.fullServerID = try innerData.decode(String.self, forKey: .fullServerID)
+        // Deal with error thrown when loading subreddit content the key display_name is not present and continue
+        do {
+           self.displayName = try innerData.decode(String.self, forKey: .displayName)
```

Comment:
does every listing contain a displayName? if so, you should make `public var displayName: String?` non optional, removing the `?`.

If it is truly optional, you should remove the `do {} catch {}` and replace with `try?`

Reply:
Not every listing contains a displayName.